src/aliceVision/sfm/README.md

```diff
@@ -0,0 +1,239 @@
+# sfm
+
+sfm is the module related to Structure from Motion.
+It handles storage of SfM related data and method to solve SfM problems (camera pose estimation, structure triangulation, bundle adjustment).
+
+
+## A generic SfM data container
```

Comment:
this should go in the sfmData module, shouldn't it?

Reply:
You're right!